docs/UserGuide.md

```diff
@@ -13,37 +13,38 @@ title: User Guide
-  [4.4.4. Viewing item in profile: `view me`](#viewing-item-in-profile-view-me) <br />
-  [4.4.5. Finding items in profile: `find me`](#finding-items-in-profile-find-me) <br />
-  [4.4.6. Listing all profile items: `list me`](#listing-all-items-in-profile-list-me) <br />
- [4.5 General](#general) <br />
-  [4.5.1. Generating matching internships: `match`](#generating-matching-internships-match) <br />
-  [4.5.2. Switching Tabs: `switch`](#switching-tabs-switch) <br />
-  [4.5.3. Viewing Help: `help`](#viewing-help-help) <br />
-  [4.5.4. Clearing all entries: `clear`](#clearing-all-entries-clear) <br />
-  [4.5.5. Exiting the program: `exit`](#exiting-the-program-exit) <br />
+ [4.1. Navigating InternHunter](#navigating-internhunter) <br />
+  [4.1.1. Switching tabs: `switch`](#switching-tabs-switch) <br />
+ [4.2. Company](#company) <br />
+  [4.2.1. Adding a company: `add com`](#adding-a-company-add-com) <br />
+  [4.2.2. Deleting a company: `delete com`](#deleting-a-company-delete-com) <br />
+  [4.2.3. Editing a company: `edit com`](#editing-a-company-edit-com) <br />
+  [4.2.4. Viewing a company: `view com`](#viewing-a-company-view-com) <br />
+  [4.2.5. Finding companies: `find com`](#finding-companies-find-com) <br />
+  [4.2.6. Listing all companies: `list com`](#listing-all-companies-list-com) <br />
+ [4.3. Internship](#internship) <br />
+  [4.3.1. Adding an internship: `add int`](#adding-an-internship-add-int) <br />
+  [4.3.2. Deleting an internship: `delete int`](#deleting-an-internship-delete-int) <br />
+  [4.3.3. Editing an internship: `edit int`](#editing-an-internship-edit-int) <br />
+ [4.4. Application](#application) <br />
+  [4.4.1. Applying for an internship: `add app`](#applying-for-an-internship-add-app) <br />
+  [4.4.2. Deleting an application: `delete app`](#deleting-an-application-delete-app) <br />
+  [4.4.3. Updating an application: `edit app`](#updating-an-application-edit-app) <br />
+  [4.4.4. Viewing an application: `view app`](#viewing-an-application-view-app) <br />
+  [4.4.5. Finding applications: `find app`](#finding-applications-find-app) <br />
+  [4.4.6. Listing all applications: `list app`](#listing-all-applications-list-app) <br />
+ [4.5. Profile](#profile) <br />
+  [4.5.1. Adding item to profile: `add me`](#adding-item-to-profile-add-me) <br />
+  [4.5.2. Deleting item in profile: `delete me`](#deleting-item-in-profile-delete-me) <br />
+  [4.5.3. Editing item in profile: `edit me`](#editing-item-in-profile-edit-me) <br />
+  [4.5.4. Viewing item in profile: `view me`](#viewing-item-in-profile-view-me) <br />
+  [4.5.5. Finding items in profile: `find me`](#finding-items-in-profile-find-me) <br />
+  [4.5.6. Listing all profile items: `list me`](#listing-all-items-in-profile-list-me) <br />
+ [4.6. General](#general) <br />
+  [4.6.1. Generating matching internships: `match`](#generating-matching-internships-match) <br />
```

Comment:
Just a quick question, are we taking `match` out of the General section? So for example we could have:
* 4.6. Generating matching internships: `match`
* 4.7. General
...
I realised our group consensus on this might not have been documented (unless I missed it out).

Reply:
Yeah I am not so sure about this also. Maybe we see how @keanecjy does it? then I will update the ToC

Reply:
Sure thing!

Reply:
@shawn-nyk Fixed in latest Pr. Confirmed with keane already.